plugins/Categories.cc

```diff
@@ -43,18 +43,14 @@ bool MuMuCategory::event_in_category_post_analyzers(const ProducersManager& prod
 };
 
 void MuMuCategory::register_cuts(CutManager& manager) {
-    manager.new_cut("fire_trigger_Mu17_Mu8", "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v*");
-    manager.new_cut("fire_trigger_Mu17_TkMu8", "HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_DZ_v*");
-    manager.new_cut("fire_trigger_IsoMu27", "HLT_IsoMu27_v*");
+    manager.new_cut("fire_trigger_MuMu", "HLT_Mu*");
 };
 
 void MuMuCategory::evaluate_cuts_post_analyzers(CutManager& manager, const ProducersManager& producers, const AnalyzersManager& analyzers) const {
     const HLTProducer& hlt = producers.get<HLTProducer>("hlt");
     for (const std::string& path: hlt.paths) 
     {
-        if (path.find("HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v") != std::string::npos) manager.pass_cut("fire_trigger_Mu17_Mu8");
-        if (path.find("HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_DZ_v") != std::string::npos) manager.pass_cut("fire_trigger_Mu17_TkMu8");
-        if (path.find("HLT_IsoMu27_v") != std::string::npos) manager.pass_cut("fire_trigger_IsoMu27");
+        if (path.find("HLT_Mu") != std::string::npos) manager.pass_cut("fire_trigger_MuMu");
```

Comment:
This condition is also true for MuEl paths, which mean an event will be in two categories at the same time (not sure it's possible?). It'd probably be easier to use a regex here, something like `HLT_Mu.*_Mu`